test/integration/rtpbroadcast.js

```diff
@@ -34,7 +35,8 @@ describe('Rtpbroadcast tests', function() {
       .then(function(session) {
         janusSession = session;
         done();
```

Comment:
When we catch exceptions, we see errors in output. When we do `finally`, we see timeout errors.